### PR TITLE
Final set of comments before submission

### PR DESCRIPTION
**Pull request by brandmaier**
Please integrate, @aaronpeikert. Thanks!

### CODE REVIEW

File: manuscript.Rmd [hunk 1]
```diff
@@ -980,6 +981,7 @@ The code for this power analysis can be found in the [online supplementary mater
 
 Monte Carlo simulations are, of course, not only applicable for this analysis method and also allow researchers to investigate other relevant properties of their analysis method beyond power [@brandmaier2018precision; @harrisonIntroductionMonteCarlo2010; @raychaudhuriIntroductionMonteCarlo2008].
 
+<!--AB this code will not be understood by most of the readers? Can we remove this? Instead keep the results text below but explain what the variables choosen_power and minn mean and that they were either set in the simulation or simulation results #necessary-->
```
Comment (aaronpeikert) [inline manuscript.Rmd:984]: Hey @brandmaier this code is hidden (echo=FALSE).

Comment (brandmaier) [reply manuscript.Rmd:984]: Hm... I rendered the document outside Docker and this showed up in the PDF. Not sure why. Anyhow, then everything is fine.

Comment (aaronpeikert) [reply manuscript.Rmd:984]: Curious and very weird that knitr would malfunction like that, maybe there was a typo or something in this particular chunks options.

Comment (brandmaier) [reply manuscript.Rmd:984]: Screenshot 2021-08-05 at 21 18 55

This is what it looks like.

Comment (aaronpeikert) [reply manuscript.Rmd:984]: True! This is a very similar code I forgot that it was still in there.

**Pull request merged**